inst/tinytest/test-app-auth-basic.R

```diff
@@ -8,15 +8,15 @@ app = ex_app("auth-basic")
 
 # Test /hello endpoint
 rq = Request$new(path = "/hello")
-rs = app$process_request(rq)
+rs = app$process_request(rq)$to_rserve()
```

Comment:
I think we must test the `Response` class fields directly. It's more clear.
We already test the `to_rserve` method in a separate test.

Reply:
Agree - need to rework this in future. I'm merging this PR as we haven't touch any logic.